src/Umbraco.Web/PublishedCache/NuCache/DataSource/BTree.cs

```diff
@@ -14,6 +15,12 @@ internal class BTree
                 CreateFile = exists ? CreatePolicy.IfNeeded : CreatePolicy.Always,
                 FileName = filepath,
 
+                // read or write but do *not* keep in memory
+                CachePolicy = CachePolicy.None,
```

Comment:
@zpqrtbnk Will merge this in but also just wanted to understand what this setting does?

Reply:
BPlusTree does more than what we need - it can act as some kind of local database, and we could read stuff from it at any time, not just read everything when the site starts. In this case, it's good to have a cache policy that keeps the MRU items in memory and the rest on disk. In our case, we load everything once and then manage our own cache = better to not have BPlusTree cache anything.